specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/apimauthorizationproviders.json

```diff
@@ -582,6 +582,69 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/authorizationProviders/{authorizationProviderId}/authorizations/{authorizationId}/confirmConsentCode": {
+      "post": {
+        "tags": [
+          "AuthorizationConfirmConsentCode"
+        ],
+        "operationId": "AuthorizationConfirmConsentCode_Post",
```

Comment:
`post` is not a good Verb in operatoinId. Suggest to improve the operationId name:
https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/openapi-authoring-automated-guidelines.md#r1001-operationidnounverb

Reply:
Updated to the verb "confirm"